### ISSUE TITLE
Midpoint Review - Content Feedback - Unified Appointment Experience, Patient Check In, Add Upcoming Appointments

### ISSUE THREAD

**Issue by KKitagawa-Bosch** [department-of-veterans-affairs/va.gov-team#89765]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** Under the heading "you're checked in" where it says "the staff can call you back any time" this sentence should have a clearer verb. Based on our discussion in the midpoint review meeting, I recommend "the staff may call you back" 
- [ ] **Must:** in the same sentence "the staff can call you back any time" the phrase "any time" should be two words, as it is functioning here as a noun
- [ ] **Must:** In the "video appointment" section, the content currently says "we'll add the link the join", but does not specify where the link will appear. Some additional content is needed so users know where to look for that link in the future. 
- [ ] **Must:** In the "video appointment" section, the guidance tells users to "click the join session button". We should avoid the word "click" as not all users will actually be clicking on buttons depending on what device they use. (https://design.va.gov/content-style-guide/neutral-and-inclusive-language#language-that-doesnt-assume-an-ability-or-device)
- [ ] **Must:** several locations in the content use straight apostrophes ('). Design system guidance asks that we always use curly apostrophes (’). I noticed this specifically in the "travel reimbursement", "video appointment", and "phone appointment" sections, but comb through to make sure there aren't other instances! 
- [ ] **Should:** In the "video appointment" section, users are told to be in a "well-lighted" room. While this is technically correct, it is a more formal/less common phrasing, and should be replaced with "well-lit" to better meet plain language standards.
- [ ] **Should:** in the "upcoming appointments" section the sentence "to find all your . . . community care health facility" should include a comma for clarity. The sentence should read "you can use our appointment tools on VA.gov, or call your VA or community care health facility." 

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by benbrasso-agile6** [comment]
@outerpress handfuls of comments on content for video appt type

**Comment by outerpress** [comment]
@KKitagawa-Bosch @laurwill re: video appointments on appointment detail pages (bullets 3-6 under Feedback)

In [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/89478#issuecomment-2261529633) we talked about moving the content that's currently in the add'l info to a separate page. 

[This design file](https://www.figma.com/design/HnExAM5SGNXzUlv9yRS87t/Details-Page-Changes?node-id=2011-2251&t=wIEtpy5daCVsbJeh-4) shows how that would look after incorporating Keegan's feedback above. Could y'all review and drop comments with any further feedback in that file?

Also cc: @mistymg because words :)

**Comment by benbrasso-agile6** [comment]
Thanks, Peter

**Comment by KKitagawa-Bosch** [comment]
@outerpress Sent a request for access to the Figma file! Tag me in when that's sent, and I'll get right on it!

**Comment by KKitagawa-Bosch** [comment]
Looking good! Bullets 3, 4, and 6 are great

In bullet 5,  there's still a straight apostrophe in "we'll" on the video appointment page, just FYI! 

Appreciate y'all!

**Comment by outerpress** [comment]
Fixed the sneaky straight apostrophe, thanks @KKitagawa-Bosch !

**Comment by outerpress** [comment]
@benbrasso-agile6 design ticket #89998 will cover the video visit work. Appointments product folks will need to prioritize this one, so I'm not sure when this will get picked up.

**Comment by benbrasso-agile6** [comment]
It's no problem. Our release in check-in will propose a workaround for our product. E.g., what we can do in full auth vs LOROTA. Will share when it's ready for review.

**Comment by benbrasso-agile6** [comment]
Blocked; waiting on https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/89876 to be resolved

**Comment by benbrasso-agile6** [comment]
think we need to keep this one open for a bit until Peter makes some movement with the unchecked items

**Comment by outerpress** [comment]
@benbrasso-agile6 #89998 isn't high priority for us so it might be a bit. Ideally we could split the unchecked items into a separate ticket since they're really not in scope for the CC scheduling work. @shiragoodman is that an option?

**Comment by benbrasso-agile6** [comment]
@shiragoodman wanted to reping on this one. Could we split out the unchecked item so this ticket can be closed?

**Comment by KKitagawa-Bosch** [comment]
@benbrasso-agile6 sorry for that delay, Shira is OOO for two weeks so I was discussing with the team before responding. 

I talked with the team, and we're thinking this is fine so long as you connect the two tickets so we can keep a paper trail of the process!

**Comment by benbrasso-agile6** [comment]
Oh, I didn't even think to check. Thanks! Yeah; will chat with Peter in a few mins about this.

**Comment by benbrasso-agile6** [comment]
@outerpress

**Comment by outerpress** [comment]
Thanks @KKitagawa-Bosch @benbrasso-agile6 ! I captured the AC from this one in #89998, feel free to add any tags to that one necessary for tracking.

**Comment by benbrasso-agile6** [comment]
Added that ticket number as a blocker. Thanks, Peter.

**Comment by outerpress** [comment]
Resolved in https://github.com/department-of-veterans-affairs/va.gov-team/issues/111533
